packages/kbn-reporting/export_types/png/png_v2.ts

```diff
@@ -122,9 +119,12 @@ export class PngExportType extends ExportType<JobParamsPNGV2, TaskPayloadPNGV2>
         return this.startDeps
           .screenshotting!.getScreenshots({
             format: 'png',
+            browserTimezone: payload.browserTimezone,
```

Comment:
Oops! `browserTimezone` was not being passed before. That could have caused PNG reports showing date values formatted in the timezone of the Kibana server rather than the end user client. It doesn't seem to be a previosly known issue.